mtp_noms_ops/apps/security/views/check.py

```diff
@@ -60,9 +65,50 @@ def get_context_data(self, **kwargs):
             {'name': self.title},
         ]
         context_data[self.object_context_key] = detail_object
-
+        context_data['related_credits'] = self._get_related_credits(api_session, context_data[self.object_context_key])
         return context_data
 
+    @staticmethod
+    def _get_related_credits(api_session, detail_object):
+        # Get the sender credits
+        sender_response = api_session.get(
+            '/senders/{sender_profile_id}/credits/?{querystring}'.format(
+                sender_profile_id=detail_object['credit']['sender_profile'],
+                querystring=urlencode([
+                    ('exclude_credit__in', detail_object['credit']['id']),
```

Comment:
I understand your point about embedding ids in request parameters, they can be a code smell under certain circumstances. But I think having the exclusion parameter explicitly encoded in the request gives us a more explicit interface that fits with existing filter logic and can be used in other use cases. I don't think moving the functionality from the API to noms-ops would make a big difference in terms of complexity, and if for some reason we had to remove the ability to bypass pagination in our API (say if we opened it up to some non-MOJ integration) then it would make things hellishly difficult

Reply:
sure let's keep `exclude_credit__in`. tho i would never be a fan of removing pagination from mtp-api